### PR TITLE
Get the body

### PR DESCRIPTION
**Pull request by Happypig375**

### CODE REVIEW

File: src/actionOutputs.ts [hunk 1]
```diff
@@ -9,8 +9,9 @@ export const actionOutputs = {
     tag_name: new ActionOutput('tag_name'),
     target_commitish: new ActionOutput('target_commitish'),
     name: new ActionOutput('name'),
+    name: new ActionOutput('body'),
```
Comment (cardinalby) [inline src/actionOutputs.ts:12]: duplicated key `name`
Should be `body`

Comment (Happypig375) [reply src/actionOutputs.ts:12]: Oh thanks

Comment (Happypig375) [reply src/actionOutputs.ts]: Fixed

**Comment by cardinalby** [comment]
available under @master, @v1, @1.0.1

**Pull request merged**